lib/tasks/populate_government_data.rake

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+namespace :app do
+  desc "Populate government data"
+  task populate_government_data: :environment do
+    PopulateBulkDataJob.perform_now
+  end
+end
```

Comment:
I can see a few issues here (namespace app, task focused on government data but the job named differently) though mostly I'm wondering why we need the task since it's one line, may as well suggest someone runs the command in the console.

Reply:
Have removed